lib/purl_finder.rb

```diff
@@ -85,7 +85,11 @@ def index_purls(params={})
       druid = get_druid_from_file_path(line)
       if !druid.blank?
         IndexingLogger.info("indexing #{druid}")
-        result = Purl.save_from_public_xml(File.dirname(line)) # pass the directory of the file containing public
+        begin
+          result = Purl.save_from_public_xml(File.dirname(line)) # pass the directory of the file containing public
+        rescue => e
+          IndexingLogger.error("An error occurred while trying to save #{druid}.  \n#{e.message} \n#{e.backtrace.inspect}")
+        end
```

Comment:
@mejackreed @lmcglohon just to be clear, does `result` become `false` if the exception is thrown? it's not clear to me that `error` gets incremented when an exception is raised from just reading the code. It looks like the spec does verify that `error` is getting set. thoughts?